winui/Kanban/Workflows.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: post
+title: Kanban Workflow Setup | SfKanban | WinUI | Syncfusion<sup>®</sup>Syncfusion<sup>®</sup>
+description: This section provides details about configuring workflows in Syncfusion<sup>®</sup> Essential Studio<sup>®</sup> WinUI Kanban (SfKanban) control, its components, and functionality.
+platform: WinUI
+control: SfKanban
+documentation: ug
+---
+
+# Workflow configuration
+
```

Comment:
To define the flow of card transitions between different states, create an instance of the `KanbanWorkflow` class and add it to the Workflows property of the `SfKanban` to define the workflow for each column. The `KanbanWorkflow` class contains the following properties to specify the source and target categories.